addon/components/cfb-form-editor/question/options.js

```diff
@@ -96,6 +98,12 @@ export default RenderComponent.extend({
       this._update();
     },
 
+    archiveRow(row) {
```

Comment:
Since this toggles the `isArchived` property I'd prefer a method name like `toggleRowArchived` to be more explicit in what this actually does.

Reply:
Done :-)